ios/Classes/TiMapCircleProxy.m

```diff
@@ -126,11 +134,16 @@ -(void)setStrokeColor:(id)value
 
 -(void)setStrokeWidth:(id)value
 {
-
     strokeWidth = [TiUtils floatValue:value];
     [self applyStrokeWidth];
 }
 
+-(void)setOpacity:(id)value
+{
+    alpha = [TiUtils floatValue:value];
+    [self applyOpacity];
+}
```

Comment:
I don't like that code-style but I do understand that you wanted to follow the way the other code did it. I may need refactor that whole module some day ... But let's keep it like it is for now.